README.md

```diff
@@ -129,134 +122,17 @@ Notice that, for communicating with the team tools, API keys may be required. It
-
-- **`layouts/`**: Nuxt layout components.
-
-  More information: [NuxtJS documentation on the _layouts_ directory](https://nuxtjs.org/docs/2.x/directory-structure/layouts)
-
-- **`mixins/`**: Shared Vue.js mixins.
-
-- **`new-content/`**: Content Markdown files included via `@nuxtjs/content`.
-
-  This directory would usually be named `content`, but that name was already in use in our project.
-
-  More information: [NuxtJS documentation on the _content_ directory](https://nuxtjs.org/docs/2.x/directory-structure/content)
-
-- **`pages/`**: The base application views and routes.
-
-  More information: [NuxtJS documentation on the _pages_ directory](https://nuxtjs.org/docs/2.x/directory-structure/pages)
-
-- **`plugins/`**: JavaScript plugins that run before instantiating the root Vue.js application.
-
-  More information: [NuxtJS documentation on the _pages_ directory](https://nuxtjs.org/docs/2.x/directory-structure/plugins)
-
-- **`static/`**: Files that will be automatically served by Nuxt and will be accessible through the project root URL.
-
-  More information: [NuxtJS documentation on the _static_ directory](https://nuxtjs.org/docs/2.x/directory-structure/static)
-
-- **`store/`**: Vuex store files.
-
-  More information: [NuxtJS documentation on the _store_ directory](https://nuxtjs.org/docs/2.x/directory-structure/store)
-
-- **`tests/`**: Jest unit tests.
-
-- **`types/`**: Additional types for non-typed libraries or global definitions.
-
-- **`nuxt.config.js`**: Main NuxtJS configuration.
-
-  More information: [NuxtJS documentation on the _nuxt.config_ file](https://nuxtjs.org/docs/2.x/directory-structure/nuxt-config)
+We follow Nuxt's [directory structure](https://nuxt.com/docs/guide/directory-structure/nuxt).
 
 ## ✏️ How to Contribute
 
-Contributions are always welcome, no matter how large or small. Before contributing, please read the [contributing guide](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md).
-
-## 🛠 Available Scripts
-
-Run a local server enabling inspector agent:
-
-```shell
-  npm run dev-debug
-```
-
-Run unit tests made with [Jest](https://jestjs.io/):
-
-```shell
-  npm run test
-```
-
-Build static version ready for production, output will be generated inside a new folder called `dist`:
-
-```shell
-  npm run build
-```
-
-Run a local server on the website's production built. Make sure you ran `npm run build` first:
-
-```shell
-  npm run start
-```
-
-Find syntax errors. We use [ESLint](https://eslint.org/):
-
-```shell
-  npm run lint
-```
-
-Autofix linter:
-
-```shell
-  npm run fix-lint
-```
+Contributions are always welcomed, no matter how large or small. Before contributing, please read the [contributing guide](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md).
 
 ## 🗓 Open backlog
 
-You can see our backlog [here](https://github.com/Qiskit/qiskit.org/projects/2).
+You can see our backlog [here](https://github.com/orgs/Qiskit/projects/10).
 
 ## 👩‍💻 Maintainers
```

Comment:
at some point, we need to update our list of Maintainers as well...

we can ignore for now since this is focused on our migration efforts